### DISCUSSION TITLE
Any guidance on animation?

### DISCUSSION THREAD

**Discussion by charleneyjh1** [twilio-labs/paste#2264] [Q&A] [answered]
Hi team, our team is open to exploring adding a bit of animation in our flow. 
We'd like to add animation to this feedback question "Did you receive an SMS?". The question would appear once they click "Copy the code" button. With animation we are hoping to achieve a more smooth and friendly UX, and make this question more noticeable. 
We were wondering what is DSYS's take on adding animation in general? Do you have any guidance around it? Are you opposed to  using an animation library?(We may or may not use an animation library, if the animation is easy, we may not need it)

Thank you! 

<img width="755" alt="Screen Shot 2022-03-03 at 5 34 28 PM" src="https://user-images.githubusercontent.com/56697620/156676570-a8cc6ada-84af-4e61-9f15-fb2a67d61603.png">

**Comment by shleewhite** [comment, marked as answer]
Hi @charleneyjh1! 

We don't have any formal guidance on animation right now, but your team can use [the Paste Animation library](https://github.com/twilio-labs/paste/tree/main/packages/paste-libraries/animation), which wraps [react-spring](https://react-spring.io/).

For accessibility, when animating new content onto the page, it is best practice to add the new content below where the user is so they will navigate to it with either a keyboard or with a screen reader as they naturally move through the page. If it gets added above where they currently are, the user will probably not know something was added and that they have to move backwards.
